_targets.R

```diff
@@ -0,0 +1,35 @@
+library(targets)
+source("code.R")
+tar_option_set(packages = c("tidyverse", "sbtools", "whisker"))
+
+list(
+  # Get the data from ScienceBase
+  tar_target(
+    model_RMSEs_csv,
```

Comment:
Great work, but it would be great to download this file to `1_fetch/out/model_RMSEs.csv` instead. Can you make a change to make sure that happens when you build this target?

Reply:
Done! Fixed in 3cf32d5.